cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+require( '../polyfill/class-list' );
+
+/**
+ * Get the sibling nodes of a dom node.
+ *
+ * @param {HTMLNode} elem - DOM element to get siblings of.
+ * @param {string} className - CSS classname.
```

Comment:
Could we make this any selector, or even none at all, to make it more robust? I could see a use like `getSiblings( navItem, [aria-expanded="true"] )` to find an open nav item at the same hierarchy level as a target nav item.